src/FileInput.js

```diff
@@ -97,7 +97,6 @@ class FileInput extends PureComponent {
  * @typedef {Object} PropTypes
  * @static
  *
- * @prop {string} name
```

Comment:
In `Input.js` we use `name` and `id`. Not sure if they apply to a `FileInput` but if they do the consistency would be nice:

https://github.com/dhis2/ui-core/blob/next/src/Input.js#L141-L142

Reply:
All right, I'm not sure what's applicable here. @Mohammer5 @HendrikThePendric?

Reply:
yes, plz add them.